src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,40 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+You can import file types which aren't natively supported by Astro. There are many **Vite** and compatible **Rollup** plugins you can use. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of Vite Documentation.
+
+### Example: YAML support
+
+The YAML (`.yml`) data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin:
+
+1. Install `@rollup/plugin-yaml`:
+
+    ```bash
+    npm install @rollup/plugin-yaml --save-dev
+    ```
+
+2. Import the plugin in your `astro.config.mjs` and add it to the Vite plugins array:
+
+    ```js title="astro.config.mjs" ins={2,5-7}
+    import { defineConfig } from 'astro/config';
+    import yaml from '@rollup/plugin-yaml';
+
+    export default defineConfig({
+      vite: {
+        plugins: [yaml()]
+      }
+    });
+    ```
+
+    :::note[Using a different plugin?]
+    Refer to your plugin's documentation to learn what configuration options it has, and how to correctly install it.
+    :::
```

Comment:
```suggestion
    :::note[every plugin is different!]
    Refer to your plugin's documentation to learn which configuration options it has, and how to correctly install it.
    :::
```
I'm trying to think of maybe a clearer title, since the asnwer to "using a different plugin?" is probably.. yes?  😅  Of alllll the plugins in the world, yes, probably using a different one!  😄

Reply:
I haven't been able to come up with any thing better:

plugin configuration
configuring plugins

Thoughts?

Reply:
The concern is that “Using a different plugin” is not clear that we mean “…from the YAML one”?

One option is just to move this content up out of the example steps and say something like this note says in the introduction after the “finding plugins” hint.

Reply:
Ooh, yes, let me look at this. Maybe where we say this is an example, something about "not everything works exactly like this."